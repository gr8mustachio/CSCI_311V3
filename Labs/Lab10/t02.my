Size of the heap is 2
Capacity of the heap is 2
0 0 
0 0 
0 0 

