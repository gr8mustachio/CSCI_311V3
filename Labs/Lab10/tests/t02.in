push 0 30
push 1 20
getSize
getCapacity
print
