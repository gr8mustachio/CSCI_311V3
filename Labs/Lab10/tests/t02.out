Size of the heap is 2
Capacity of the heap is 2
20 30 
1 0 
1 0 

